WcaOnRails/app/helpers/application_helper.rb

```diff
@@ -137,13 +137,18 @@ def users_to_sentence(users, include_email: false)
     end.xss_aware_to_sentence
   end
 
+  def year_option_tags(selected_year: nil, exclude_future: false)
```

Comment:
I think this should default to `true` in order to stay equivalent to the previous code. It's a logic inversion of the former, am I right?

Reply:
Ah, you're totally right, thank you!